### PR TITLE
Integrate RN Nightly 10/1 (223e98cc4)

### PR DESCRIPTION
**Pull request by TatianaKapos**
## Description
Integration
Commit Range: https://github.com/facebook/react-native/compare/1747f57c6...223e98cc4

## Notable Commits
- Libraries/DevToolsSettings -> private/reactdevtools/ReactDevToolsSettings (https://github.com/facebook/react-native/pull/46662)
- [New implementation of PerformanceObserver and related APIs (fixes #45122)](https://github.com/facebook/react-native/pull/45206)
- [Switch Babel parsing from legacy Flow plugin to hermes-parser #46696](https://github.com/facebook/react-native/pull/46696)
- [Switch to hermes-parser in eslint-* package tests #46699](https://github.com/facebook/react-native/pull/46699)
- [Refactor FabricMountingManager to enable reuse of IntBuffer methods #46600](https://github.com/facebook/react-native/pull/46600)
- [Add a new playground component to rntester #46632](https://github.com/facebook/react-native/pull/46632)
- [Require Flow types in all react-native source files #45786](https://github.com/facebook/react-native/pull/45786)

## Issues Created
-  https://github.com/microsoft/react-native-windows/issues/13939 - Upstream PR made :)

### CODE REVIEW

File: packages/@rnw-scripts/babel-react-native-config/babel.config.js [hunk 1]
```diff
@@ -11,7 +11,6 @@ module.exports = () => ({
     'module:@react-native/babel-preset',
   ],
   plugins: [
-    "babel-plugin-syntax-hermes-parser",
```
Comment (marlenecota) [inline packages/@rnw-scripts/babel-react-native-config/babel.config.js:14]: we still need this until the jest-preset fix is published

Comment (tido64) [reply packages/@rnw-scripts/babel-react-native-config/babel.config.js]: This plugin was added to `@react-native/babel-preset` in https://github.com/facebook/react-native/commit/1387f521fdd8f187eab7a4a6a05d4d75a96b4f88. We should be able to remove this line if we also bump the preset to 0.77-nightly: https://github.com/TatianaKapos/react-native-windows/blob/a459663cd72ab4d151915bb19461847a9f26ea70/packages/%40rnw-scripts/babel-react-native-config/package.json#L14-L19

Comment (TatianaKapos) [reply packages/@rnw-scripts/babel-react-native-config/babel.config.js:14]: that seemed to fix it! Thanks Tommy! :)

**Review by marlenecota — APPROVED** [review]

**Pull request merged**